cl_dll/hud_strafeguide.cpp

```diff
@@ -0,0 +1,157 @@
+#include <complex>
+
+#include "hud.h"
+#include "cl_util.h"
+
+#include "pm_defs.h"
+#include "pm_movevars.h"
+
+enum border {
+	RED_GREEN,
+	GREEN_WHITE,
+	WHITE_GREEN,
+	GREEN_RED
+};
+
+const float pi = std::acos(-1.);
+
+int CHudStrafeGuide::Init()
+{
+	m_iFlags = HUD_ACTIVE;
+
+	hud_strafeguide = CVAR_CREATE("hud_strafeguide", "0", FCVAR_ARCHIVE);
+	hud_strafeguide_fov = CVAR_CREATE("hud_strafeguide_fov", "140", FCVAR_ARCHIVE);
```

Comment:
Shouldn't it just use the current FOV?

Reply:
I thought that people might want to change it. It controls the overall horizontal scale, and can be quite small.You can see in the image in the PR that at 300-ish speeds the green region is already pretty tiny with my FOV of 140.
Looking at it again, it's obvious that `hud_strafeguide_fov 0` should refer to using `default_fov`, and should be the default choice.
I know you don't like having lots of cvars, so I thought I'd start by putting in every one I could think of and then letting you decide which ones you want to keep :^) It's fine if you want to get rid of this one.

Reply:
I'll need to check how it looks myself in-game to see how much this "zooming in" is needed. Speaking of, if this is indeed for "zooming in" then perhaps it should be called zoom and have values like 1, 1.5, 2, etc?